pkg/controller/installation/core_controller.go

```diff
@@ -456,6 +456,14 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, err
 	}
 
+	// A status is needed at this point for operator scorecard tests.
+	// status.variant is written later but for some tests the reconciliation
+	// does not get to that point.
+	instance.Status = operator.InstallationStatus{}
+	if err = r.client.Status().Update(ctx, instance); err != nil {
```

Comment:
Makes sense during the first run. But on subsequent runthroughs of the reconcilliation loop, won't this nuke existing statuses? Or is that not a problem because we mainly write status to the tigerastatus CR instead of the installation CR?

FWIW, I ran into this same gap while working on a different feature and this is what I came up with: https://github.com/ozdanborne/operator/commit/df37bb48d34c9543aa6cefd6f622eeb55e776df1

Reply:
What I had was wrong. But thanks for sharing that diff... I've written my fixed commit so that it might merge with your future changes a bit more easily.